Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```
@@ -265,6 +265,10 @@
     - trayScanner
     - AccessConfiguratorUniversal
     - Multitool
+    - WelderExperimental
+    - PowerDrill
+    - GeigerCounter
+    - JawsOfLife
```

Wait, whoops, this needs to be commented for Omu.
```suggestion
    - WelderExperimental # Omu Edit - More Admin Tools
    - PowerDrill # Omu Edit - More Admin Tools
    - GeigerCounter # Omu Edit - More Admin Tools
    - JawsOfLife # Omu Edit - More Admin Tools
```